app.js

```diff
@@ -21,78 +20,41 @@ client.login(process.env.DISCORD_TOKEN);
 
 client.on('messageCreate', async function (message)
 {
-    if (message.content.includes('https://twitter.com/'))
-    {        
-        let name = '';
-        let guild = client.guilds.cache.get(message.guildId);
-        guild.members
-            .fetch(message.author)
-            .then(data => name = data.nickname + ' (' + data.user.username + ')');
+    const ADRESS_TO_CHANGE_TO = "vxtwitter.com";
+    let messageContent = message?.content ?? "";
+    let repostTweet = false;
 
-        let tweetId = message.content.match(/\/status\/(\d+)/)[1];
-        if (await TweetContainsVideo(tweetId))
-        {
-            message.delete();
-            message.channel.send(name + ' posted: ' + message.content.replace('twitter.com', 'vxtwitter.com'));
-        }
+    // switch to vxtwitter
+
+    if(messageContent.includes('https://twitter.com/')) 
+    {
+        messageContent = messageContent.replace('twitter.com', ADRESS_TO_CHANGE_TO);
+        repostTweet = true;
+    } 
+    else if (messageContent.includes('https://x.com/')) 
+    {
+        messageContent = messageContent.replace('x.com', ADRESS_TO_CHANGE_TO);
+        repostTweet = true;
     }
-    else if (message.content.includes('https://x.com/'))
+
+    // should be a !repost tweet + return statment but I can't test in this environment
+    if(repostTweet) 
     {
-        let name = '';
+        // strip tracking link
+        if (messageContent.match(/\?t=/gm) != null) 
+        {
+            messageContent = messageContent.match(/.+?(?=\?t=)/gm)?.[0] ?? messageContent;
```

Comment:
I think there's an extra period here before the [0]

Haven't tested yet but it looks off

Reply:
![image](https://github.com/BenFort/twotter-bot/assets/32877602/a0dcf6fb-d080-4e90-8197-f332d76cc36e)

Reply:
Still seems weird, remove it please

Reply:
Ok nevermind, I guess that's the syntax if you're including the null check